logp/config.go

```diff
@@ -41,7 +41,6 @@ type Config struct {
 	environment Environment
 	addCaller   bool // Adds package and line number info to messages.
 	development bool // Controls how DPanic behaves.
-	logsPath    string
```

Comment:
This was unused.

Reply:
Yes, it is a leftover from the migration of the package. Thanks.